lib/routes/cwc.js

```diff
@@ -0,0 +1,59 @@
+var senateOfficeData = require('../data/scwc-offices.json');
+var houseOfficeData = require('../data/cwc-offices.json');
+var data = require('../data');
+var xml = require('xml2js');
+var _ = require('lodash');
+
+function addData(payload){
+  return data.SetData(payload, payload.email);
+}
+
+function constructPayload(req, type){
+    // using the express.raw() middleware sets the req.body to a buffer that we can convert back into a string
+    var xmlString = req.body.toString();
+    return xml.parseStringPromise(xmlString).then(function(parsed){
+        var email = _.get(parsed, 'CWC.Constituent[0].Email[0]') || 'no-email';
+        return {
+            cwcType: type,
+            email: email,
+            hasApiKey: req.query.apikey ? 'yes' : 'no',
```

Comment:
do you want us to send a stub API key to jane casey for consistency?

Reply:
we should send something, probably shouldnt be the live key